legacy-sdk/whirlpool/tests/integration/v2-ix/initialize/set_reward_emissions_v2.test.ts

```diff
@@ -126,10 +132,23 @@ describe("set_reward_emissions_v2", () => {
             .addSigner(configKeypairs.rewardEmissionsSuperAuthorityKeypair)
             .buildAndExecute();
 
-          whirlpool = (await fetcher.getPool(
-            poolInitInfo.whirlpoolPda.publicKey,
-            IGNORE_CACHE,
-          )) as WhirlpoolData;
+          whirlpool = await (async () => {
```

Comment:
curious - do we need the same for lines 111-119? i see it's doing something similar, i.e. setting emissions to a non-zero value first